Frameworks/CoreText/CTLine.mm

```diff
@@ -283,16 +283,44 @@ CGRect CTLineGetImageBounds(CTLineRef line, CGContextRef context) {
 }
 
 /**
- @Status Caveat
- @Notes Values only reflect the typographical maximums for the font - string is only evaulated
-        for width
+ @Status Interoperable
+ @Notes
 */
 double CTLineGetTypographicBounds(CTLineRef lineRef, CGFloat* ascent, CGFloat* descent, CGFloat* leading) {
-    if (!lineRef) {
+    _CTLine* line = static_cast<_CTLine*>(lineRef);
+
+    if (!line || [line->_runs count] == 0) {
         return 0;
     }
 
-    _CTLine* line = static_cast<_CTLine*>(lineRef);
+    // Created with impossible values FLT_MIN which signify they need to be populated
```

Comment:
>// Created with impossible values FLT_MIN which signify they need to be populated [](start = 4, length = 81)

I'm not so sure about this... 
1) what's your reasoning here that FLT_MIN is an impossible, rather than merely improbable value?
2) it's not super evident that this was the intended purpose of FLT_MIN in DWriteWrapper